serialization/Schemas/spdx-v3.json

```diff
@@ -0,0 +1,1061 @@
+          }
+        }
+      }
+    },
+    "Snippet": {
+      "title": "Snippet",
+      "type": "object",
+      "additionalProperties": false,
+      "required": [
+        "location"
+      ],
+      "maxProperties": 100,
+      "properties": {
+        "location": {
+          "$ref": "#/definitions/SnippetLocation",
+          "description": ""
+        },
+        "snippetPurpose": {
+          "type": "array",
+          "description": "",
+          "minItems": 1,
+          "items": {
+            "$ref": "#/definitions/SoftwarePurpose",
+            "description": ""
+          }
+        },
+        "originator": {
+          "type": "array",
+          "description": "<-Artifact:  Link(Identity)",
+          "minItems": 1,
+          "items": {
+            "$ref": "#/definitions/Element",
+            "description": "<-Artifact:  Link(Identity)"
+          }
+        }
+      }
+    },
+    "File": {
+      "title": "File",
+      "type": "object",
+      "additionalProperties": false,
+      "maxProperties": 100,
+      "properties": {
+        "contentType": {
+          "$ref": "#/definitions/MediaType",
+          "description": ""
+        },
+        "filePurpose": {
+          "type": "array",
+          "description": "",
+          "minItems": 1,
+          "items": {
+            "$ref": "#/definitions/SoftwarePurpose",
+            "description": ""
+          }
+        },
+        "originator": {
+          "type": "array",
+          "description": "<-Artifact:  Link(Identity)",
+          "minItems": 1,
+          "items": {
+            "$ref": "#/definitions/Element",
+            "description": "<-Artifact:  Link(Identity)"
+          }
+        }
+      }
+    },
+    "Package": {
+      "title": "Package",
+      "type": "object",
+      "additionalProperties": false,
+      "maxProperties": 100,
+      "properties": {
+        "packagePurpose": {
+          "type": "array",
+          "description": "",
+          "minItems": 1,
+          "items": {
+            "$ref": "#/definitions/SoftwarePurpose",
+            "description": ""
+          }
+        },
+        "downloadLocation": {
+          "$ref": "#/definitions/URL",
+          "description": ""
+        },
+        "homePage": {
+          "$ref": "#/definitions/URL",
+          "description": ""
+        },
+        "originator": {
+          "type": "array",
+          "description": "<-Artifact:  Link(Identity)",
+          "minItems": 1,
+          "items": {
+            "$ref": "#/definitions/Element",
+            "description": "<-Artifact:  Link(Identity)"
+          }
+        }
+      }
+    },
+    "Sbom": {
+      "title": "Sbom",
+      "type": "object",
+      "additionalProperties": false,
+      "required": [
+        "element"
+      ],
+      "maxProperties": 100,
+      "properties": {
+        "element": {
+          "type": "array",
+          "description": "<-Collection",
```

Comment:
What do you mean by the "<-" before "Collection"? I assumed you are referring to the description of a Collection in the schema, but there is none.

Reply:
The comments in ElementType show class inheritance.  Collection was a class in some previous version of the model, but with the proposed simplification SpdxDocument would be the only class inheriting from the abstract ElementCollection, or ElementCollection would disappear entirely if there are no use cases for something other than a Payload being a collection of Element values.